tests/schema.sql

```diff
@@ -1,4 +1,4 @@
-CREATE DATABASE IF NOT EXISTS `phpstan-dba`;
+CREATE DATABASE IF NOT EXISTS `phpstan_dba`;
```

Comment:
How did this work?

Reply:
it seems we created a database with this schema file, but used a different one :).

but you already figured that one out as far as I can see. good catch